Validation/RecoVertex/python/pvRecoSequence_cff.py

```diff
@@ -30,6 +31,7 @@
        ),
        vertexCollections = cms.VPSet(
               [cms.PSet(label=cms.string(""),
+                        chi2cutoff = cms.double(2.5),
```

Comment:
Shouldn't this be 3.0 to preserve the old behaviour? (same for the other `chi2cutoff` below)